### PR TITLE
Change the date format

### PR DESCRIPTION
**Pull request by Diama1**

### CODE REVIEW

File: client/src/elm/Utils/NominalDate.elm [hunk 1]
```diff
@@ -129,7 +129,7 @@ renderDate language date =
     )
         ++ " "
         ++ month
-        ++ ", "
+        ++ " "
```
Comment (anvmn) [inline client/src/elm/Utils/NominalDate.elm:132]: No need for all these ++. Lets make it `++ " month " ++`.

**Review by anvmn — CHANGES_REQUESTED** [review]

File: client/src/elm/Utils/NominalDate.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Utils/NominalDate.elm:132]: @Diama1 ^^

Comment (Diama1) [reply client/src/elm/Utils/NominalDate.elm:132]: > No need for all these ++. Lets make it `++ " month " ++`.

@anvmn but month is not a string, it is a variable

Comment (anvmn) [reply client/src/elm/Utils/NominalDate.elm:132]: @Diama1 right, my mistake.
Sorry.

**Review by anvmn — APPROVED** [review]

**Pull request merged**